tests/integration_test/core_mqtt_system_test.c

```diff
@@ -226,7 +226,7 @@
  * PUBLISH message and ack responses for QoS 1 and QoS 2 communications
  * with the broker.
  */
-#define MQTT_PROCESS_LOOP_TIMEOUT_MS          ( 700U )
+#define MQTT_PROCESS_LOOP_TIMEOUT_MS          ( 10000U )
```

Comment:
Increasing the wait from 0.7 to 10 seconds every time the process loop is called seems like it would waste time in a lot of cases. If the only reason for this is to ensure that an ack is received, then wouldn't this be more efficient?
```c
void waitForAck( bool *flag ) {
    int counter = 0;
    TEST_ASSERT_FALSE( *flag );
    while( !(*flag) && ( counter++ < 10 ) ) {
        /* Event callback will set flag when receiving appropriate ack. This will
         * wait for at most TRANSPORT_SEND_RECV_TIMEOUT_MS. */
        TEST_ASSERT_EQUAL( MqttSuccess, MQTT_ProcessLoop( &context, 0 ) );
    }
    TEST_ASSERT_TRUE( *flag );
}
  
```

Reply:
Sure thanks